.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -40,13 +40,19 @@ const workflowURL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOS
 /**
  * @param {String} deployer
  * @param {String} deployVerb
+ * @param {String} prTitle
  * @returns {String}
  */
-function getDeployMessage(deployer, deployVerb) {
+function getDeployMessage(deployer, deployVerb, prTitle) {
     let message = `🚀 [${deployVerb}](${workflowURL}) to ${isProd ? 'production' : 'staging'}`;
     message += ` by @${deployer} in version: ${version} 🚀`;
     message += `\n\n platform | result \n ---|--- \n🤖 android 🤖|${androidResult} \n🖥 desktop 🖥|${desktopResult}`;
     message += `\n🍎 iOS 🍎|${iOSResult} \n🕸 web 🕸|${webResult}`;
+
+    if (deployVerb === 'Cherry-picked' && !(/no qa/gi).test(prTitle)) {
+        message += '\n\nThe PR title did not include [No QA], so this CP requires QA @Expensify/applauseleads';
```

Comment:
NAB but I prefer this message:

```suggestion
        message += '\n\n@Expensify/applauseleads please QA this PR and check it off on the [deploy checklist](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3AStagingDeployCash) if it passes.';
```